src/views/HomeView.vue

```diff
@@ -166,7 +166,8 @@ export default {
       return this.$vuetify.display.mdAndDown ? 'small' : 'medium';
     },
   },
-  async mounted() {
+  async created() {
+    localStorage.setItem('returningUser', true);
```

Comment:
Do you need to set this every time, or should we only be setting it if it doesn't exist? I don't mind either way, this is less complicated and it likely doesn't matter. What this does have, though, is a bug for when localStorage doesn't exist or errors out.  Check out the implementation in `src/components/CreateGameDialog.vue` via `setRankedPreference`. It uses try/catch to avoid any errors.

What I recommend is creating a utility to set/get a `localStorage` value and use that as a cuttle implementation of LS with graceful fallback in the event of an error. It should be used both when you check the value, and when you set the value.

Reply:
Where should these functions live?

Reply:
Then the created hook here would dispatch the auth store module's `setIsReturningUser()` action here, and the navigation guard in `src/router.js` would get it's value using the `getIsReturningUser()` store value